cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIDynamoDBSinkTest.java

```diff
@@ -27,28 +29,132 @@
  */
 public class NGSIDynamoDBSinkTest {
     
+    /**
+     * [NGSIDynamoDBSink.configure] -------- When not configured, not mandatory parameters get default values.
+     */
     @Test
-    public void dummyTest() {
-    } // dummyTest
-    
-    private NGSIBatch createBatch(long recvTimeTs, String service, String servicePath, String destination,
-            NotifyContextRequest.ContextElement contextElement) {
-        NGSIEvent groupedEvent = new NGSIEvent(recvTimeTs, service, servicePath, destination, null,
-            contextElement);
-        NGSIBatch batch = new NGSIBatch();
-        batch.addEvent(destination, groupedEvent);
-        return batch;
-    } // createBatch
+    public void testConfigureDefaults() {
+        System.out.println(getTestTraceHead("[NGSIDynamoDBSink.configure]")
+                + "-------- When not configured, not mandatory parameters get default values");
+        String accessKeyId = "my_access_key";
+        String attrPersistence = null;
+        String batchSize = null;
+        String batchTimeout = null;
+        String batchTTL = null;
+        String dataModel = null;
+        String enableGrouping = null;
+        String enableLowercase = null;
+        String region = null;
+        String secretAccessKey = "my_secret_access_key";
+        NGSIDynamoDBSink sink = new NGSIDynamoDBSink();
+        sink.configure(createContext(accessKeyId, attrPersistence, batchSize, batchTimeout, batchTTL, dataModel,
+                enableGrouping, enableLowercase, region, secretAccessKey));
+        
+        try {
+            assertTrue(sink.getRowAttrPersistence());
+            System.out.println(getTestTraceHead("[NGSIDynamoDBSink.configure]")
+                    + "-  OK  - 'attr_persisnce=row' configured by default");
+        } catch (AssertionError e) {
+            System.out.println(getTestTraceHead("[NGSIDynamoDBSink.configure]")
+                    + "- FAIL - 'attr_persisnce=row' not configured by default");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1162/commits/e8604df05d9f9d9033f2810069d8c8cc76633587